ddp-workspace/projects/ddp-angio/src/app/app.module.ts

```diff
@@ -60,8 +60,8 @@ tkCfg.mailingListDialogUrl = 'updates';
 tkCfg.phone = '857-500-6264';
 tkCfg.infoEmail = 'info@ascproject.org';
 tkCfg.dataEmail = 'data@ascproject.org';
-tkCfg.twitterAccountId = 'ASCaProject';
-tkCfg.facebookGroupId = 'groups/1556795987968214';
+tkCfg.twitterAccountId = 'https://twitter.com/count_me_in';
```

Comment:
It should be just `count_me_in`.

Otherwise there will be a wrong URL there:
![image](https://user-images.githubusercontent.com/7396837/169006100-85dc5855-7f8f-42e1-9d65-6a4a258528ec.png)